.github/workflows/kind-e2e.yaml

```diff
@@ -49,6 +49,17 @@ jobs:
         - registry-name: registry.local
           registry-port: 5000
           registry-url: registry.local:5000
+        # Run the v1alpha1 conformance suite with alpha tests enabled.
+        # This is explicitly added since we don't want to run every single matrix element twice.
+        - k8s-version: v1.19.1
+          test-suite: ./test/conformance/api/v1alpha1
+          test-flags: "--enable-alpha"
+          kind-version: v0.9.0
+          kind-image-sha: sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600
+          kingress: istio
+          registry-name: registry.local
+          registry-port: 5000
+          registry-url: registry.local:5000
```

Comment:
You don't need to copy this part down

Reply:
This shouldn't be part of the matrix, we should fix this...

Reply:
oh, you're right: it's completely static isn't it.. can we just inline it or make it an `env`?

Reply:
will rebase this on https://github.com/knative/serving/pull/9838 once it lands, thanks @mattmoor